openstudio_measure_tester.gemspec

```diff
@@ -32,10 +32,10 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'git', '~> 1.13.0'
   spec.add_dependency 'minitest', '~> 5.14.0'
   spec.add_dependency 'minitest-reporters', '~> 1.4.2'
-  spec.add_dependency 'parser', '3.2.2.2'
+  spec.add_dependency 'parser', '>= 3.2.2.2'
   spec.add_dependency 'rake', '~> 13.0'
 
-  spec.add_development_dependency 'rubocop', '1.50'
+  spec.add_development_dependency 'rubocop', '>= 1.50.0'
```

Comment:
The gem ecosystem is fragile since everything has to align, I would refrain from touching that stuff unless you're 1) absolutely sure of what you're doing and 2) working with all gem repos in parallel